pinot-connectors/pinot-spark-2-connector/pom.xml

```diff
@@ -35,6 +35,45 @@
   </properties>
 
   <profiles>
+    <profile>
+      <id>build-shaded-jar</id>
+      <activation>
```

Comment:
I think you want to use `active-profiles` here, see https://maven.apache.org/guides/introduction/introduction-to-profiles.html